io/GDALReader.cpp

```diff
@@ -100,7 +100,16 @@ QuickInfo GDALReader::inspect()
         throwError("Couldn't open raster file '" + m_filename + "'.");
 
     qi.m_pointCount = m_raster->width() * m_raster->height();
-    qi.m_bounds = m_raster->bounds();
+
+    auto p = std::find(m_bandIds.begin(), m_bandIds.end(), Dimension::Id::Z);
+
+    int nBand(1);
```

Comment:
Probably should document the fact that the first band is taken to be Z if there is no header specified or if Z isn't present.

Reply:
> the first band is taken to be Z

only for the QuickInfo. I wasn't sure what to do otherwise.